cmd/main.go

```diff
@@ -892,3 +909,21 @@ func startPPROF(log *logrus.Logger) {
 		}
 	}
 }
+
+func runOpenshiftReleaseSyncerIfNeeded(kubeAPIMode, enableOpenshiftReleaseSyncer bool, ReleaseSourcesHandler releasesources.ReleaseSourcesHandler, log logrus.FieldLogger) *thread.Thread {
+	if !kubeAPIMode && enableOpenshiftReleaseSyncer {
+		go ReleaseSourcesHandler.SyncReleaseImages()
```

Comment:
Would it make sense to run synchronously on startup? Not sure if this data is necessary on startup or not

Reply:
I think a run of this code can take some time, I don't know if it is better to keep the service waiting. Is there a serious con in running it in the background ?

Reply:
Potentially, the con of running in background is that if we need those versions for the application to work, it won't work until the task is done

Reply:
@rccrdpccl we need those versions for the application to run? I'm not sure about it
we might miss the "dynamic" versions for some time, but it shouldn't make the application crash or something like that